Lesson_1/HelloName_Lesson/HelloName_Lesson/Lesson4.cs

```diff
@@ -0,0 +1,162 @@
+﻿using System;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Text.RegularExpressions;
+
+namespace HelloName_Lesson
+{
+	internal class Lesson4
+	{
+        /// <summary>
+        /// Заменить в строке все вхождения 'test' на 'testing'. Удалить из текста все символы, являющиеся цифрами.
+        /// </summary>
+        
+        public static void HomeTask4()
+		{
+            string NewString = "test and 1test 231231 best test";
```

Comment:
name of the local variable starting from lowercase should be newString
change please for all variables in the code

Reply:
but anyway, I've changed everything to the unified style